config/prod/recover-input-shared-bucket.yaml

```diff
@@ -0,0 +1,37 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+#### 
+# NOTE:
+# This is the first bucket in the ingress pipeline for RECOVER project. We expect to receive data from Care Evolution in this bucket.
+# This would be the raw version of the data, no QC, no QA - just all the data dump from Care Evolution
+# Both Care Evolution and Sage have access to this bucket
+####
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-input-shared-bucket"
+stack_tags:
+  Department: "SysBio"
+  Project: "RECOVER"
+  OwnerEmail: 'synapse-service-sysbio-recover-data-storage-01@sagebase.org'
+  # This is the shared account (credentials shared on Last Pass) for RECOVER
+  CostCenter: "MGH RECOVER / 122500"
+parameters:
```

Comment:
Several of these parameters don't exist in [the template](https://github.com/Sage-Bionetworks/aws-infra/blob/v0.5.0/templates/S3/synapse-external-bucket.j2) (`Department`, `Project`, `OwnerEmail`, and `SynapseUserName`) and should be removed here.